### PR TITLE
Fix search path in update script

### PR DESCRIPTION
**Pull request by cevian**

### CODE REVIEW

**Review by jgpruitt — APPROVED** [review]

File: sql/promscale--0.1.2--0.1.3-beta.sql [hunk 1]
```diff
@@ -40,11 +40,11 @@ CREATE AGGREGATE @extschema@.vector_selector(
     sample_time timestamptz,
     sample_value DOUBLE PRECISION)
 (
-    sfunc = vector_selector_transition,
+    sfunc = @extschema@.vector_selector_transition,
```
Comment (JamesGuthrie) [inline sql/promscale--0.1.2--0.1.3-beta.sql:43]: It looks like we have the same pattern of issue in `promscale--0.1.3.beta.sql`

**Review by JamesGuthrie — CHANGES_REQUESTED** [review]

File: sql/promscale--0.1.2--0.1.3-beta.sql (hunk 1 above)
Comment (sumerman) [reply sql/promscale--0.1.2--0.1.3-beta.sql:43]: What about `extension_sql!` in `lib.rs`? Do we have to fix it there as well, or does it not affect the SQL that ships?

Comment (JamesGuthrie) [reply sql/promscale--0.1.2--0.1.3-beta.sql:43]: That doesn't affect the SQL which ships, so I think that we can skip it for now.

File: sql/promscale--0.3.0--0.3.1.sql [hunk 2]
```diff
@@ -0,0 +1,49 @@
+CREATE OR REPLACE FUNCTION @extschema@.label_find_key_equal(key_to_match prom_api.label_key, pat prom_api.pattern)
```
Comment (sumerman) [inline sql/promscale--0.3.0--0.3.1.sql:1]: 1. Shouldn't we assert or set ownership of these functions? 
2. Does it matter whether a type was fully specified or not when aggregates and operators were defined? If it does, they may require a similar treatment.

**Review by sumerman — CHANGES_REQUESTED** [review]
I believe it's missing `sql/promscale--0.3.2.sql` symlink.

**Comment by cevian** [comment]
@sumerman good catch. Fixed!

**Review by sumerman — APPROVED** [review]

**Review by JamesGuthrie — APPROVED** [review]

**Pull request merged**